php/class-post-type.php

```diff
@@ -849,4 +862,20 @@ public function filter_out_settings_if_removed_in_metabox( $content ) {
 
 		return $content;
 	}
+
+	/**
+	 * Get preview url query vars.
+	 *
+	 * @param int $post_id Post id.
+	 * @return array $preview_url_query_vars Preview url query vars.
+	 */
+	public function get_preview_url_query_vars( $post_id ) {
+		$preview_url_query_vars = array();
+
+		if ( $post_id ) {
+			$preview_url_query_vars = (array) get_post_meta( $post_id, '_preview_url_query_vars', true );
```

Comment:
If there is no postmeta on the changeset post, where `get_post_meta()` returns `false`, this will result in `$preview_url_query_vars === array( false )`. 

So this should probably be:

```php
$preview_url_query_vars = get_post_meta( $post_id, '_preview_url_query_vars', true );
if ( ! is_array( $preview_url_query_vars ) ) {
    $preview_url_query_vars = array();
}
```